config.frontend.test.yaml

```diff
@@ -11,8 +11,6 @@ default_project: &default_project
                     host: https://parsoid-beta.wmflabs.org
                     grace_ttl: 1000000
                     php_host: https://en.wikipedia.beta.wmflabs.org/w/rest.php
```

Comment:
let's remove `php_host` and just use `host`? we don't need 2 of these any longer.